test/pages/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,90 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Landing page renders correctly 1`] = `
+<Layout
+  description="Welcome to Node.js!"
+  title="Node.js Official website"
+  withBg={true}
+>
+  <MainSection
+    content={
+      Object {
+        "desc": "Tacos raw denim palo santo, squid 90s occupy sustainable leggings locavore before they sold out chambray gastropub synth. Cornhole lomo dreamcatcher celiac enamel pin, yr lyft master cleanse meggings.",
+        "downloadButton": "Download Node (LTS)",
+        "downloadCurrent": "Get Current",
+        "learnNode": "Learn Node",
+        "subDownloadMessage": "What’s new",
+        "title": "The power of JavaScript minus the browser",
+      }
+    }
+  />
+  <SandboxSection />
+  <FeaturesSection
+    content={
+      Object {
+        "featureList": Array [
+          Object {
+            "image": "test-file-stub",
+            "text": "Feat #1 Cornhole lomo dreamcatcher celiac enamel pin, yr lyft master cleanse meggings.",
+          },
+          Object {
+            "image": "test-file-stub",
+            "text": "Feat #2 Tacos raw denim palo santo, squid 90 occupy sustainable leggings locavore before they sold out chambray gastropub synth.",
+          },
+          Object {
+            "image": "test-file-stub",
+            "text": "Feat #3 Lorem ipsum dolor amet pug vape +1 poke pour-over kitsch tacos meh. ",
+          },
+        ],
+      }
+    }
+  />
+  <CommunitySection
+    content={
+      Object {
+        "desc": "We’ll never share your information and always respect your inbox - quality content only, we promise.",
```

Comment:
I love this assertion. I'd like to get a strict definition from the Foundation of what this means – what _their_ usage of the list is – so we can actually detail how we define this when someone will inevitably ask.

Reply:
same reason as here https://github.com/nodejs/nodejs.dev/pull/324#discussion_r331869223